tutorials/tmva/TMVA_RNN_Classification_PyTorch.py

```diff
@@ -0,0 +1,308 @@
+import ROOT
```

Comment:
The code should be integrated in the tutorial we are adding in the https://github.com/root-project/root/pull/10442